w3-aggregation.md

```diff
@@ -165,99 +305,166 @@ Broker MUST issue a signed receipt to acknowledge the received request. Issued r
     "join": { "/": "bafy...dequeue" }
   },
   "meta": {},
-  "iss": "did:web:spade.storage",
+  "iss": "did:key:agg...",
   "prf": []
 }
 ```
 
-See [`offer/arrange`](#offerarrange) section to see the subsequent task.
+When piece request to be added is dequeued, aggregator should invoke `piece/add` to include it in an aggregate.
 
-### `aggregate/get`
+> `did:key:agg...` invokes capability from `did:key:agg...`
 
-A Storefront principal can query state of accepted aggregate by invoking `aggregate/get` capability. See [schema](#aggregateget-schema).
+```json
+{
+  "iss": "did:key:agg...",
+  "aud": "did:key:agg...",
+  "att": [{
+    "with": "did:key:agg...",
+    "can": "piece/add",
+    "nb": {
+      "piece": { "/": "commitment...car" }, /* commitment proof for piece */
+      "group": "did:web:free.web3.storage", /* grouping for joining segments together into an aggregate */
+    }
+  }],
+  "prf": [],
+  "sig": "..."
+}
+```
```

Comment:
@Gozala note that now we cannot use `with` to derive who is the tenant. We probably need to add a `storefrontId` that then we can map to tenantId for aggregate offer